shared/src/main/java/me/neznamy/tab/shared/proxy/ProxyTabPlayer.java

```diff
@@ -169,8 +169,11 @@ public void sendPluginMessage(@NotNull OutgoingMessage message) {
 
     @Override
     public boolean isVanished() {
-        for (VanishIntegration i : VanishIntegration.getHandlers()) {
-            if (i.isVanished(this)) return true;
+        if (!VanishIntegration.getHandlers().isEmpty()) {
```

Comment:
Checking the same condition in `isVanished` is unnecessary, and I can't think of a valid reason why an integration should return `false` if the player is actually vanished. However, I believe it's better to respect the integration's choice in this case